contracts/rewards/StableTRIStaking.sol

```diff
@@ -58,12 +58,14 @@ contract StableTRIStaking is Ownable, ERC20 {
     /// @notice The deposit fee, scaled to `DEPOSIT_FEE_PERCENT_PRECISION`
     uint256 public depositFeePercent;
     /// @notice The precision of `depositFeePercent`
-    uint256 public DEPOSIT_FEE_PERCENT_PRECISION;
+    uint256 public immutable DEPOSIT_FEE_PERCENT_PRECISION;
+    /// @notice The max deposit fee value, scaled to `DEPOSIT_FEE_PERCENT_PRECISION`
+    uint256 public MAX_DEPOSIT_FEE_PERCENT = 5e17;
```

Comment:
make immutable

Reply:
I tried, it was complaining, will try again 
maybe can make immutable and init in constructor

Reply:
<img width="546" alt="Screenshot 2022-05-27 at 11 31 00" src="https://user-images.githubusercontent.com/3833339/170731251-65cb4a3c-e30d-4f1f-9aa1-284b16e7163e.png">
yeah I did try, I'll init in constructor but make it immutable